lte/gateway/python/integ_tests/s1aptests/test_attach_service_with_multi_pdns_and_bearers.py

```diff
@@ -0,0 +1,415 @@
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import unittest
+import time
+
+import gpp_types
+import s1ap_types
+import s1ap_wrapper
+from integ_tests.s1aptests.s1ap_utils import SpgwUtil
+from integ_tests.s1aptests.s1ap_utils import SessionManagerUtil
+from integ_tests.s1aptests.ovs.rest_api import get_datapath, get_flows
+import ipaddress
+
+
+class TestAttachServiceWithMultiPdnsAndBearers(unittest.TestCase):
+    SPGW_TABLE = 0
+    GTP_PORT = 32768
+    LOCAL_PORT = "LOCAL"
+    default_ip = None
+
+    def setUp(self):
+        self._s1ap_wrapper = s1ap_wrapper.TestWrapper()
+        self._sessionManager_util = SessionManagerUtil()
+        self._spgw_util = SpgwUtil()
+
+    def tearDown(self):
+        self._s1ap_wrapper.cleanup()
+
+    def _verify_flow_rules(
```

Comment:
Let's make this a library function in s1ap utilities. You can pass a list of flow match rules as the argument to be verified rather than providing ad hoc parameters and defining different "verify_flow_rules" functions for different test cases.